src/java.base/share/classes/jdk/internal/logger/BootstrapLogger.java

```diff
@@ -964,6 +983,18 @@ static Logger getLogger(LazyLoggerAccessor accessor) {
         }
     }
 
+    private static void ensureClassInitialized(Class<?> c) {
+        try {
+            MethodHandles.lookup().ensureInitialized(c);
```

Comment:
Hello Sean, this `BootstrapLogger` appears to be used/applicable even before VM is booted. Would introducing the use of `MethodHandles` at this stage (when the VM perhaps hasn't yet booted) have any other implications (for example, startup performance etc...)?